test/passing/tests/break_separators-after.ml.ref

```diff
@@ -371,7 +370,8 @@ let g () =
        hhhhhhhhhh |] ->
       fooooooooo
 
-let () = match x with _, (* line 1 line 2 *) Some _ -> x
+let () = match x with _, (* line 1 line 2 *)
+                         Some _ -> x
```

Comment:
This is a regression.

Reply:
This change was introduced in intermediate state of this PR and goes away if the test file is reverted. The bug exists but is not related to this PR.